sources/packages/backend/apps/api/src/route-controllers/assessment/_tests_/e2e/assessment.students.controller.getAssessmentAwardDetails.e2e-spec.ts

```diff
@@ -521,6 +615,9 @@ describe("AssessmentStudentsController(e2e)-getAssessmentAwardDetails", () => {
       {
         offeringIntensity: OfferingIntensity.fullTime,
         applicationStatus: ApplicationStatus.Completed,
+        firstDisbursementInitialValues: {
+          disbursementScheduleStatus: DisbursementScheduleStatus.Sent,
+        },
```

Comment:
date sent is not added for part-time test.

Reply:
I added to one part-time and one full-time.